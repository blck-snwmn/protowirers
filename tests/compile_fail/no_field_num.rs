use protowirers::*;

#[derive(Proto)]
struct Sample {
    #[def(def_type = "int32")]
    s: u32,
}

fn main() {}
