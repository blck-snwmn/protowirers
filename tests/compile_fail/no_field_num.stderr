error: invalid num of meta in #[def(...)]. got=1, expected=2
 --> $DIR/no_field_num.rs:6:5
  |
6 |     s: u32,
  |     ^
